Testscripts/Linux/verify_vmbus_heartbeat_properties.sh

```diff
@@ -19,6 +19,14 @@
 # Source constants file and initialize most common variables
 UtilsInit
 
+# Install python 2.7 if missing
+python_path=$(which python)
+LogMsg "Checked the system Python path: $python_path"
+if [[ $? -gt 0 ]]; then
+	install_package python2
+	ln -s /usr/bin/python2.7 /usr/bin/python
+fi
```

Comment:
I am not sure if the package name is python2 in different distro, the same situation for python2.7

Reply:
This is why I tested randomly in 4 different distro versions. If there is anything else corner-case, will fix